diff --git a/src/pwmio.py b/src/pwmio.py
--- a/src/pwmio.py
+++ b/src/pwmio.py
@@ -20,9 +20,6 @@ from adafruit_blinka.importing import raise_for_missing_platform_dependency
 try:
-    if detector.board.any_raspberry_pi_5_board:
-        from adafruit_blinka.microcontroller.generic_linux.lgpio_pwmout import PWMOut
-    elif detector.board.any_raspberry_pi:
-        # Pi 4 or lower
-        from adafruit_blinka.microcontroller.generic_linux.rpi_gpio_pwmout import PWMOut
+    if detector.board.any_raspberry_pi:
+        from adafruit_blinka_raspberry_pi_pwm import PWMOut
     elif detector.board.any_bananapi:
         from adafruit_blinka.microcontroller.generic_linux.sysfs_pwmout import PWMOut
     elif detector.board.any_coral_board:
